server/src/http_shared_missing_scores.go

```diff
@@ -65,13 +65,14 @@ func httpSharedMissingScores(c *gin.Context) {
 		}
 	}
 
+	lastIndex := len(playerNames) - 1
 	data := &TemplateData{ // nolint: exhaustivestruct
 		Title:               "Missing Scores",
-		NamesTitle:          "Missing Scores for [" + strings.Join(playerNames, ", ") + "]",
+		NamesTitle:          "Missing Scores for " + strings.Join(playerNames[:lastIndex], ", ") + " and " + playerNames[lastIndex],
```

Comment:
isnt this bugged when there is only one player

Reply:
![image](https://user-images.githubusercontent.com/2622043/146614336-e7833537-615c-4976-8fb6-901c7bec063f.png)

Reply:
ok, nevermind then